Rethinking_2/Chp_07.ipynb

```diff
@@ -0,0 +1,3517 @@
+{
```

Comment:
<p>Yes, this should be done using OLS.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Switched to OLS